django/forms/forms.py

```diff
@@ -210,17 +210,20 @@ def _html_output(self, normal_row, error_row, row_ender, help_text_html, errors_
                 if css_classes:
                     html_class_attr = ' class="%s"' % css_classes
 
+                extra_attrs = {'aria-describedby': f'{bf.id_for_label}_helptext'}
                 if errors_on_separate_row and bf_errors:
                     output.append(error_row % str(bf_errors))
+                    extra_attrs['aria-describedby'] += f' {bf.id_for_label}_errorlist'
 
+                bf.field.widget.attrs.update(extra_attrs)
```

Comment:
Question: would there be any risk of this causing an issue if Django users out there are already manually setting an `aria-describedby` on some of their fields?